app/preprints/-components/submit/review/template.hbs

```diff
@@ -1,67 +1,117 @@
+                <h4>
+                    {{t 'preprints.submit.step-five.preprint-title'}}
+                </h4>
+                <div local-class='display'>
+                    {{this.preprint.title}}
                 </div>
             </div>
+            <div local-class='content-container'>
+                <Preprints::-Components::PreprintAbstract @preprint={{this.preprint}} />
+            </div>
+            <hr>
         </div>
-        <div local-class='content-container'>
-            <div local-class='title'>
-                {{t 'preprints.submit.step-five.preprint-title'}}
+        <div local-class='step-container'>
+            <h3 local-class='title'
+                data-test-title-step-two
+            >
+                {{t 'preprints.submit.step-two.title'}}
+            </h3>
+            <div local-class='content-container'>
+                <div local-class='title'>
+                    {{t 'preprints.submit.step-five.contributors'}}
+                </div>
+                <div local-class='display'>
+                    <ContributorList
+                        @model={{this.preprint}}
+                        @shouldTruncate={{false}}
+                        @shouldLinkUsers={{true}}
+                    />
+                </div>
             </div>
-            <div local-class='display'>
-                {{this.preprint.title}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintLicense @preprint={{this.preprint}} />
+                </div>
             </div>
-        </div>
-        <div local-class='content-container'>
-            <div local-class='title'>
-                {{t 'preprints.submit.step-five.description'}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintDiscipline @subjects={{this.subjects}} />
+                </div>
             </div>
-            <div local-class='display'>
-                {{this.preprint.description}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintTag @preprint={{this.preprint}} />
+                </div>
             </div>
+            <div local-class='content-container'>
+                <h4>
+                    {{t 'preprints.submit.step-five.publication-doi'}}
+                </h4>
+                <div local-class='display'>
+                    {{this.preprint.preprintDoiUrl}}
+                </div>
+            </div>
+            <div local-class='content-container'>
+                <h4>
+                    {{t 'preprints.submit.step-five.publication-year'}}
+                </h4>
+                <div local-class='display'>
+                    {{moment-format this.preprint.originalPublicationDate 'YYYY-MM-DD'}}
```

Comment:
Does this field display what is entered on the metadata page too? I realize the [text-input is bound to `Preprint.publicationYear`](https://github.com/CenterForOpenScience/ember-osf-web/blob/feature/preprints-phase-2/app/preprints/-components/submit/metadata/template.hbs#L109) which isn't a property on the preprint model. I think this line on the review page is correct, but the input on the metadata page should be bound to `Preprint. originalPublicationDate`